File: easy/1038.jewels-and-stones/Solution.py
class Solution:
    """
    @param J: the types of stones that are jewels
    @param S: representing the stones you have
    @return: how many of the stones you have are also jewels
    """
    def find_jewels(J, S):
        count = 0
        for j in J:
            for s in S:
                if j == s:
                    count += 1
        return count
  
    
    def numJewelsInStones(self, J, S):
        # Write your code here
        count = 0
        for c_j in J:
            for c_s in S:
                if c_s == c_j:
                    count += 1
        return count

File: easy/1038.jewels-and-stones/Solution.java
public class Solution {
    /**
     * @param J: the types of stones that are jewels
     * @param S: representing the stones you have
     * @return: how many of the stones you have are also jewels
     */
    public int numJewelsInStones(String J, String S) {
        // Write your code here
        int count = 0;
        for (int j=0;j<J.length();j++){
            for(int s=0;s<S.length();s++){
                if (J.charAt(j) == S.charAt(s)){
                    count++;
                }
            }
        }
        return count;
    }
}